src/runtime/extended-access/html-templates.js

```diff
@@ -190,6 +194,18 @@ export const REGWALL_HTML = `
       opacity: 1 !important;
     }
   }
+`;
+
+/**
+ * HTML for the metering regwall dialog, where users can sign in with Google.
+ * The script creates a dialog based on this HTML.
+ *
+ * The HTML includes an iframe that loads the Google Sign-In button.
+ * This iframe can live on a different origin.
+ */
+export const REGWALL_HTML = `
```

Comment:
Would it also be beneficial to tag the HTML strings in this file as html?

Reply:
Looks like `styled-components` just covers CSS syntax highlighting